pkg/koyeb/apps_get.go

```diff
@@ -77,8 +78,8 @@ func formatDomains(domains []koyeb.Domain, max int) string {
 
 	data, err := json.Marshal(domainNames)
 	if err != nil {
-		// We're supposed to marshal a list of strings. If this fails, we can fairly panic
-		panic(err)
+		// We're supposed to marshal a list of strings. If this fails, we can fairly fatal
+		log.Fatal("Could not marshal domain names as json")
```

Comment:
Maybe print the error so if this occurred on a client, we could know why ?

Reply:
Added!